src/System.Globalization/tests/CompareInfo/CompareInfoCompare.cs

```diff
@@ -561,6 +513,18 @@ public void Test148()
         [Fact]
         public void Test156() { Test(CultureInfo.InvariantCulture, new string('a', 5555), new string('a', 5555), 0, CompareOptions.None); }
 
+        [Fact]
```

Comment:
For a separate PR, at some point it'd be nice to clean all of these up with a theory.